src/main/java/com/google/sps/DataServlet.java

```diff
@@ -58,7 +58,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     // Initialize variables if any are null. Ideally should all be null or none
     // should be null
     if (currDataGraph == null && originalDataGraph == null) {
+      // PROTO Data structure:
+      // Parse the contents of graph.txt into a proto Graph object, and extract
+      // information from the proto object into a map. This is used to store the proto
+      // input and isn't updated with mutations.
 
+      /*
+       * The below code is used to read a graph specified in textproto form
+       */
+      // InputStreamReader graphReader = new
```

Comment:
What are these commented out lines?

Reply:
The code is different for reading a textproto file (like the one Hao gave us) and reading a binary proto file like we are using for testing right now.

Reply:
I think the reason we kept it in there was so we could switch between running the two graphs (the small one and the one that Hao gave us)!

Reply:
Ah ok. It's fine to keep both now but I wonder if you could perhaps modify the graph.txt file to be a textproto file instead so that reading the files uses the same functionality?

Reply:
Updated our tests files to be textproto :)